bin/entrypoint

```diff
@@ -2,4 +2,8 @@
 
 set -e
 
-exec pensieve run "$@"
+if [ "$1" = "export_json" ]; then
```

Comment:
This is needed for calling the JSON export from Airflow see https://github.com/mozilla/telemetry-airflow/pull/1067/files